packages/desktop-gui/src/prompts/prompts.jsx

```diff
@@ -0,0 +1,362 @@
+        element: arrowElement,
+      },
+    }, {
+      name: 'offset',
+      options: {
+        offset: [-100, 0],
+      },
+    }],
+  })
+
+  if (!isOpen) return null
+
+  return (
+    <div className={`popper prompt ${className}`} ref={setPopperElement} style={popperStyles.popper} {...popperAttributes.popper}>
+      {children}
+      <div className='arrow' ref={setArrowElement} style={popperStyles.arrow} />
+    </div>
+  )
+})
+
+@observer
+class CIPrompt1 extends Component {
+  slug = 'ci1'
+  utm_medium = 'CI Prompt 1'
+
+  ciProviders = [
+    {
+      name: 'Circle CI',
+      icon: CircleCI,
+      link: {
+        url: 'https://on.cypress.io/setup-ci-circleci',
+        params: {
+          utm_medium: this.utm_medium,
+          utm_campaign: 'Circle',
+        },
+      },
+    },
+    {
+      name: 'GitHub Actions',
+      icon: GitHubActions,
+      link: {
+        url: 'https://on.cypress.io/github-actions',
+        params: {
+          utm_medium: this.utm_medium,
+          utm_campaign: 'GitHub',
+        },
+      },
+    },
+    {
+      name: 'Bitbucket',
+      icon: Bitbucket,
+      link: {
+        url: 'https://on.cypress.io/bitbucket-pipelines',
+        params: {
+          utm_medium: this.utm_medium,
+          utm_campaign: 'Bitbucket',
+        },
+      },
+    },
+    {
+      name: 'GitLab CI/CD',
+      icon: GitLab,
+      link: {
+        url: 'https://on.cypress.io/gitlab-ci',
+        params: {
+          utm_medium: this.utm_medium,
+          utm_campaign: 'GitLab',
+        },
+      },
+    },
+    {
+      name: 'AWS CodeBuild',
+      icon: AWSCodeBuild,
+      link: {
+        url: 'https://on.cypress.io/aws-codebuild',
+        params: {
+          utm_medium: this.utm_medium,
+          utm_campaign: 'AWS',
+        },
+      },
+    },
+  ]
+
+  _close = () => {
+    this.props.prompts.closePrompt(this.slug)
+    ipc.setPromptShown(this.slug)
+  }
+
+  _openProviderLink = (link) => {
+    ipc.externalOpen(link)
+  }
+
+  _seeOther = () => {
+    ipc.externalOpen({
+      url: 'https://on.cypress.io/setup-ci',
+      params: {
+        utm_medium: this.utm_medium,
+        utm_campaign: 'Other',
+      },
+    })
+  }
+
+  _viewMore = () => {
+    ipc.externalOpen({
+      url: 'https://on.cypress.io/ci',
+      params: {
+        utm_medium: this.utm_medium,
+        utm_campaign: 'Learn More',
+      },
+    })
+  }
+
+  render () {
+    const { prompts, referenceElement } = this.props
+
+    return (
+      <Prompt
+        isOpen={prompts[this.slug]}
+        referenceElement={referenceElement}
+        className='prompt-ci1'
+      >
+        <div className='prompt-body'>
+          <button className='btn btn-link close' onClick={this._close}>
+            <i className='fas fa-times' />
+          </button>
+          <div className='text-content'>
+            <h2>Setting up CI</h2>
+            <p>We've created these guides to help you optimize how you're running tests in CI.</p>
```

Comment:
In terms of copy, this would read better if this were more user focused. No one cares what 'we did', they care what they can do.

```suggestion
            <p>Optimize how you run tests in CI by following our guides.</p>
```